qiskit/transpiler/__init__.py

```diff
@@ -155,15 +228,599 @@
+for lowering multi-qubit (3+) operations down to a series of one- and two-qubit operations.  As this is
+the first stage run, its input is a fully abstract circuit.  The ``init`` stage must be able to
+handle custom user-defined gates, and all the high-level abstract circuit-description objects, such
+as :class:`.AnnotatedOperation`.
+
+The output of the ``init`` stage is an abstract circuit that contains only one- and two-qubit
+operations.
+
+When writing :ref:`stage plugins <transpiler-preset-stage-plugins>`, the entry point for ``init`` is
+``qiskit.transpiler.init``.  The built-in plugins are:
+
+.. list-table::
+    :header-rows: 1
+
+    * - Method
+      - Summary
+
+    * - :ref:`default <transpiler-preset-stage-init-default>`
+      - Built-in unrolling of multi-qubit operations and abstract optimizations.
+
+
+.. _transpiler-preset-stage-init-default:
+
+Built-in ``default`` plugin
+...........................
+
+At optimization level 0, no abstract optimization is done.  The default plugin simply "unrolls"
+operations with more than three qubits by accessing their hierarchical
+:class:`~.circuit.Instruction.definition` fields.
+
+At optimization levels 1 and above, the default plugin also does simple cancellation of adjacent
+inverse gates, such as two back-to-back ``cx`` gates.
+
+At optimization levels 2 and 3, the default plugin enables a much wider range of abstract
+optimizations.  This includes:
+
+* "Virtual permutation elision" (see :class:`.ElidePermutations`), where explicit
+  permutation-inducing operations are removed and instead effected as remapping of virtual qubits.
+* Analysis of the commutation structure of the IR to find pairs of gates that can be canceled out.
+* Numerical splitting of two-qubit operations that can be expressed as a series of separable
+  one-qubit operations.
+* Removal of imperceivable operations, such as tiny-angle Pauli rotations and diagonal operations
+  immediately preceding measurements.
+
+.. _transpiler-preset-stage-layout:
+
+Layout stage
+------------
+
+.. seealso::
+    `Layout stage explanation <https://docs.quantum.ibm.com/guides/transpiler-stages#layout-stage>`__
+        Higher-level user-facing explanation of the layout stage in the IBM Quantum guide.
+
+The layout stage is responsible for making an initial mapping between the virtual qubits of the
+input circuit, and the hardware qubits of the target.  This includes expanding the input circuit
+with explicit ancillas so it has as many qubits as the target has, and rewriting all operations in
+terms of hardware qubits.  You may also see this problem called the "placement" problem in other
+toolkits or literature.
+
+The layout stage must set the properties ``layout`` and ``original_qubit_indices`` in the pipeline's
+:class:`.PropertySet`.
+
+.. note::
+
+    All built-in plugins for the layout stage will defer to an explicit layout selected using the
```

Comment:
I think that "defer" might be difficult to grasp for non-natives... what about?

```suggestion
    All built-in plugins for the layout stage will give priority to an explicit layout selected using the
```

Reply:
Done in 828756cb50.  I'll defer (give priority) to either preference, since I _am_ a native.